### DISCUSSION TITLE
Any way to use Bootstrap Email in PHP?

### DISCUSSION THREAD

**Discussion by TinglyTube** [bootstrap-email/bootstrap-email#88] [Help]
Hi,

We use Bootstrap 4 for the design of our website and have been wanting to switch to prettier looking emails transactional emails (We currently use simple HTML text with our logo up top) and found Bootstrap Email a few weeks ago and instantly loved it... however we use PHP and it seems Bootstrap Email doesn't have any way to use it with a PHP application, is there any kind of workaround to use Bootstrap Email with a PHP application??

Many thanks.

**Comment by stuyam** [comment]
My suggestion is to compile your emails using the editor at https://editor.bootstrapemail.com and save the precompiled files in version control so you work on those files and have the application use the actual compiled versions.

However there is also a new version being released soon (currently in alpha phase) that supports command line support, which may be easier for some people to integrate into their workflows: https://v1.bootstrapemail.com/docs/setup#command-line. It would allow you to have your html files in a folder and compile them into a subfolder that the application could use. This is still a work in progress so open to input on how it could be easier to use with something like a PHP app or what could make it easier.

**Reply by DemigodCode** [reply]
Hey @stuyam,

I've just finished a bootstrap email port to PHP. If it's okay for you, I will release it to the package manager of PHP.

**Reply by stuyam** [reply]
As long as it is clear that it isn't the _same_ bootstrap email and it is a community implementation because I just don't want like issues opened on this repo etc from that or have people think that they will be perfectly in sync with the documentation etc. I have also thought about packaging it as a binary with ruby so it could be run in any environment like PHP via the cli and write a PHP wrapper around it.

**Reply by artus70** [reply]
> I've just finished a bootstrap email port to PHP. If it's okay for you, I will release it to the package manager of PHP.

Hi DemigodCode, is this available anywhere?

**Reply by DemigodCode** [reply]
https://packagist.org/packages/antibodies-online/bootstrap-email

Have fun.

**Reply by artus70** [reply]
thx!
